### ISSUE TITLE
'z' debugger command works incorrect

### ISSUE THREAD

**Issue by lab313ru** [tonioni/WinUAE#119]
```
case 'z':
			skipaddr_start = nextpc;
			skipaddr_doskip = 1;
			do_skip = 1;
			exception_debugging = 1;
			return true;
```
in case of branches, or returns skipaddr_start will point at the next (in listing) instruction, but not at actual next address.
This case should be rewritten to skip branches, jumps, and returns and execute them as the regular step.

**Comment by tonioni** [comment]
I don't think it is a bug or working incorrectly (This was done long before I even knew about UAE), it probably was easiest way to implement it, debugger does not need to know anything about current instruction, and probable reason for having both 't' and 'z'. (It is automatic for me to choose between t and z depending on current instruction..)

Of course it would be nice to have more intelligent debugger, perhaps someday...
